components/Modal/index.d.ts

```diff
@@ -27,14 +24,14 @@ export interface ModalProps {
   };
 }
 
-export default class Modal extends Component<ModalProps> {
-  static Header: ModalHeader;
```

Comment:
Is "ModalHeader" still used in some scenario? If not, remove the type

Reply:
i agree with a remotion